jaeger-spark-dependencies-test/src/main/java/io/jaegertracing/spark/dependencies/test/TracersGenerator.java

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright 2017 The Jaeger Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.jaegertracing.spark.dependencies.test;
+
+import brave.Tracing;
+import brave.sampler.Sampler;
+import com.uber.jaeger.Tracer;
+import com.uber.jaeger.Tracer.Builder;
+import com.uber.jaeger.exceptions.SenderException;
+import com.uber.jaeger.metrics.Metrics;
+import com.uber.jaeger.metrics.NullStatsReporter;
+import com.uber.jaeger.metrics.StatsFactoryImpl;
+import com.uber.jaeger.reporters.RemoteReporter;
+import com.uber.jaeger.samplers.ConstSampler;
+import com.uber.jaeger.senders.HttpSender;
+import io.jaegertracing.spark.dependencies.test.tree.TracingWrapper;
+import io.jaegertracing.spark.dependencies.test.tree.TracingWrapper.JaegerWrapper;
+import io.jaegertracing.spark.dependencies.test.tree.TracingWrapper.ZipkinWrapper;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import zipkin.Span;
+import zipkin.reporter.AsyncReporter;
+import zipkin.reporter.Encoding;
+import zipkin.reporter.Sender;
+import zipkin.reporter.okhttp3.OkHttpSender;
+
+/**
+ * @author Pavol Loffay
+ */
+public class TracersGenerator {
+
+  public static class Tuple<A, B> {
```

Comment:
I think you do have a `Tuple` class somewhere else, don't you?

Reply:
in scala, This module does not depend on it.